ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/query/confirmation.query.hbs

```diff
@@ -34,7 +38,7 @@
         </button>
         <button class="confirmation-new is-negative">
```

Comment:
I know you didn't change this, but this button isn't negative? It looks very strange to have the "Continue with this action" button styled this way.